common/cosmwasm-smart-contracts/mixnet-contract/src/mixnode.rs

```diff
@@ -427,20 +427,31 @@ impl MixNodeBond {
     pub fn estimate_reward(
         &self,
         params: &RewardParams,
-    ) -> Result<(u64, u64, u64), MixnetContractError> {
+    ) -> Result<(u64, u64, u64, u64, u64), MixnetContractError> {
```

Comment:
I think we reached a point where those should probably get extracted to some separate struct, returning a tuple with 5 values seem to be rather confusing : )